images/base/files/usr/local/bin/entrypoint

```diff
@@ -232,6 +232,39 @@ enable_network_magic(){
   fi
 }
 
+configure_cgroup_root(){
+  if [ "${CGROUP_ROOT}" == "" ]; then
+    return
+  fi
+
+  echo "INFO: configuring cgroup root ${CGROUP_ROOT}"
+
+  CURRENT_CGROUP=$(grep systemd /proc/self/cgroup | cut -d: -f3)
```

Comment:
IIRC this mechanism was not very robust. I'm a bit concerned about this.

Reply:
What did you do instead before?

Reply:
I think this was the bit I was trying to sort out last and didn't get it solved yet, but doing some experiments suggested it might be problematic, I'm having difficulty recalling exactly.

Reply:
Alright, I'll see what I can figure out!

Reply:
Actually looks like we only do this for detecting the CGROUP_SUBSYSTEMS.. so this gets solved by rolling this stuff into the loop in the other method.